src/controllers/carpools.ts

```diff
@@ -0,0 +1,194 @@
+/// <reference path="../../typings/express-session/express-session.d.ts" />
```

Comment:
We have multiple user controller files, how about I just rename it so something more specific than carpools?